app/controllers/devise_token_auth/registrations_controller.rb

```diff
@@ -46,7 +46,9 @@ def create
               client_config: params[:config_name],
               redirect_url: @redirect_url
             })
-          else
+          end
+
+          if !@resource.respond_to?(:active_for_authentication?) || @resource.active_for_authentication?
```

Comment:
Move `!@resource.respond_to?(:active_for_authentication?) || @resource.active_for_authentication?` to a method here like `active_for_authentication?` because this controller's method is huge